README.md

```diff
@@ -80,58 +50,285 @@ nextflow info
 nextflow self-update
 ```
 
-Dockerfiles for building images are provided for each tool in the pipeline in the tool directory, `tools/<tool name>/Dockerfile`:
-- fastqc 
-- multiqc
-- trimmomatic
-- bwa
-- samtools
-- gatk
-- bcftools
-- snpeff
+### Pull Singularity images
+[Read more about Singularity here.](https://docs.sylabs.io/guides/3.2/user-guide/cli/singularity_pull.html)
+ 
+Dockerfiles for building images are provided for each tool in the pipeline in the tool directory, `tools/<tool name>/Dockerfile`. Docker images have been pre-built and are hosted on quay.io at the following links:
+
+- fastqc https://quay.io/repository/ohsu-comp-bio/fastqc
+- multiqc https://quay.io/repository/ohsu-comp-bio/multiqc
+- trimmomatic https://quay.io/repository/ohsu-comp-bio/trimmomatic
+- bwa https://quay.io/repository/ohsu-comp-bio/bwa 
+- samtools https://quay.io/repository/ohsu-comp-bio/samtools
+- gatk https://quay.io/repository/ohsu-comp-bio/gatk
+- bcftools https://quay.io/repository/ohsu-comp-bio/bcftools
+- snpeff https://quay.io/repository/ohsu-comp-bio/snpeff
+
+Because Docker cannot be easily used on Exacloud, images can be pulled using Singularity, which converts them to .sif images. To pull images:
+
+```
+## Load singularity module 
+module load /etc/modulefiles/singularity/current
+
+## set singularity cache directory variable; can live anywhere but gscratch is recommended.
+export SINGULARITY_CACHEDIR=/home/exacloud/gscratch/CEDAR/<user>/singularity/cache
```

Comment:
Let's try to abstract out specific directory names, eventually these instructions will be used by people not a OHSU, and the exacloud paths won't apply

Reply:
fixed! generalized instructions for singularity use